skrub/_dataframe/_test_utils.py

```diff
@@ -1,6 +1,33 @@
+import pandas as pd
+import pandas.testing
+
+try:
+    import polars as pl
+    import polars.testing
+except ImportError:
+    pass
+
+from .._dispatch import dispatch
+
+
```

Comment:
`is_module_pandas` is actually used nowhere
`is_module_polars` is used to raise `xfails` in some tests, it was just a convenience

Reply:
ok I guess once we have updated most tests we won't need most of those xfails anymore, and with the new fixture it will just be `df_module.name == 'polars'`, and at that point we can remove these functions